adanet/core/ensemble_builder_test.py

```diff
@@ -471,7 +475,7 @@ def _mixture_weights_train_op_fn(loss, var_list):
         self.assertAllEqual(want_replay_indices,
                             ensemble_spec.architecture.replay_indices)
 
-      with tf_compat.v1.Session(graph=g).as_default() as sess:
+      with tf_compat.v1.Session(graph=g, config=CONFIG).as_default() as sess:
```

Comment:
Instead of hardcoding the CONFIG here, we should take `self.config` which returns the `tf.estimator.RunConfig` that the user creates the estimator with. That way they have more control over these, as `RunConfig` has a `session_config` which is a tf.ConfigProto. 

So you can replace this with `config=self.config.session_config` here and below.

Reply:
Okay, I will change it, thanks for the suggestion